src/ansi-c/goto_check_c.h

```diff
@@ -47,7 +47,9 @@ void goto_check_c(
   "(retain-trivial-checks)"                                                    \
   "(error-label):"                                                             \
   "(no-assertions)(no-assumptions)"                                            \
-  "(assert-to-assume)"
+  "(assert-to-assume)"                                                         \
+  "(no-bounds-check)(no-pointer-check)(no-signed-overflow-check)"              \
+  "(no-pointer-primitive-check)(no-undefined-shift-check)"
```

Comment:
Missing `"no-div-by-zero-check`?

Reply:
Yeah, this was missed. I've added it now.